### DISCUSSION TITLE
Get verified on Flathub

### DISCUSSION THREAD

**Discussion by failafail8** [Floorp-Projects/Floorp#565] [Feature Request]
could you have the application "verified" on flathub.org ?

#https://flathub.org/about :
_I'm publishing an app on Flathub. How do I get it verified?
First, log in to Flathub.
Click the "Developer Settings" button under the app you want to verify.
At the top of the page, find the "Setup Verification" section. The 
instructions there will walk you through the verification process._

The process is pretty simple ( i think a file or line to add in the github repo)
And it would improve the trust factor by a lot .
**thank you !**

**Comment by creeper-0910** [comment]
I'll try.

**Comment by creeper-0910** [comment]
Floorp verified by Flathub!
